diff --git a/eval.py b/eval.py
index 34379c3..a5bb39d 100644
--- a/eval.py
+++ b/eval.py
@@ -53,7 +53,7 @@ from labml_helpers.device import DeviceConfigs
 from labml_helpers.module import Module
 
 import matplotlib
-matplotlib.use('Agg')  # 设置为 'Agg' 后端
+# matplotlib.use('Agg')  # 设置为 'Agg' 后端
 import matplotlib.pyplot as plt
 
 class GeneratorResNet(Module):
@@ -717,9 +717,9 @@ def plot_image(img: torch.Tensor, save_path='save.jpg'):
     # We don't need axes
     plt.axis('off')
     # Save the figure
-    plt.savefig(save_path)
+    plt.savefig(save_path, bbox_inches='tight')
     # Display
-    # plt.show()
+    plt.show()
 
     
     # Clear the current figure to free memory
@@ -922,6 +922,10 @@ def original_evaluate(image_path):
     for dir_path in dirs_to_create:
         shutil.copy(image_path, dir_path)
 
+    # 获取原始图片尺寸
+    original_img = Image.open(image_path)
+    original_size = original_img.size  # (width, height)
+
     # Start the experiment
     with experiment.start():
         # Image transformations
@@ -958,4 +962,4 @@ def original_evaluate(image_path):
 if __name__ == '__main__':
     # train()
     # evaluate(0)
-    original_evaluate()
+    original_evaluate('test.jpg')
diff --git a/save.jpg b/save.jpg
index c352a28..92640a6 100644
Binary files a/save.jpg and b/save.jpg differ