diff --git a/eval.py b/eval.py
index 34379c3..6005f49 100644
--- a/eval.py
+++ b/eval.py
@@ -53,7 +53,7 @@ from labml_helpers.device import DeviceConfigs
 from labml_helpers.module import Module
 
 import matplotlib
-matplotlib.use('Agg')  # 设置为 'Agg' 后端
+# matplotlib.use('Agg')  # 设置为 'Agg' 后端
 import matplotlib.pyplot as plt
 
 class GeneratorResNet(Module):
@@ -719,7 +719,7 @@ def plot_image(img: torch.Tensor, save_path='save.jpg'):
     # Save the figure
     plt.savefig(save_path)
     # Display
-    # plt.show()
+    plt.show()
 
     
     # Clear the current figure to free memory
@@ -958,4 +958,4 @@ def original_evaluate(image_path):
 if __name__ == '__main__':
     # train()
     # evaluate(0)
-    original_evaluate()
+    original_evaluate('test.jpg')
diff --git a/save.jpg b/save.jpg
index c352a28..dad4b6b 100644
Binary files a/save.jpg and b/save.jpg differ